pkgs/applications/networking/cluster/temporal-cli/default.nix

```diff
@@ -1,35 +1,79 @@
-{ lib, fetchFromGitHub, buildGoModule, testers, temporal-cli }:
+{ lib, fetchFromGitHub, buildGoModule, installShellFiles, symlinkJoin }:
 
-buildGoModule rec {
-  pname = "temporal-cli";
-  version = "1.18.0";
+let
+  temporal-cli = buildGoModule rec {
+    pname = "temporal-cli";
+    version = "0.8.0";
 
-  src = fetchFromGitHub {
-    owner = "temporalio";
-    repo = "tctl";
-    rev = "v${version}";
-    hash = "sha256-LcBKkx3mcDOrGT6yJx98CSgxbwskqGPWqOzHWOu6cig=";
+    src = fetchFromGitHub {
+      owner = "temporalio";
+      repo = "cli";
+      rev = "v${version}";
+      hash = "sha256-yQnFw3uYGKrTevGFVZNgkWwKCCWiGy0qwJJOmnMpTJQ=";
+    };
+
+    vendorHash = "sha256-ld59ADHnlgsCA2mzVhdq6Vb2aa9rApvFxs3NpHiCKxo=";
+
+    nativeBuildInputs = [ installShellFiles ];
+
+    excludedPackages = [ "./cmd/docgen" ];
+
+    ldflags = [ "-s" "-w" ];
```

Comment:
Also needs `"-X=github.com/temporalio/cli/headers.Version=${version}"`
for certain commands to function e.g `temporal workflow describe`